src/Components/EventTab/EventTab.module.scss

```diff
@@ -0,0 +1,150 @@
+.container {
+  margin: 0 0;
+  padding: 3rem 0;
+  margin-bottom: -1rem;
+  display: block;
+  background-color: #15131a;
+  height: fit-content;
+
+  .head {
+    padding: 1rem 0;
+    font-family: var(--srijan4-font-2);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    color: white;
+    font-size: 150%;
+    width: 100%;
+    gap: 7vw;
+
+    @media screen and (max-width: 650px) {
+      display: flex;
+      flex-direction: column;
+      font-size: 110%;
+    }
+
+    h1 {
+      height: fit-content;
+      font-weight: 600;
+    }
+
+    span {
+      font-weight: 600;
+      background: var(--srijan4-gradient-10);
+      background-clip: text;
+      -webkit-text-fill-color: transparent;
+      -webkit-background-clip: text;
```

Comment:
use class or id selectors instead of HTML element selectors

Reply:
FIX: suggested changes resolved
- spacing problem fixed
- data extracted from single json
- removing the use of html selectors